articles/azure/azs-how-create-S2S-VPN-azurestack-to-azure.md

```diff
@@ -3,8 +3,8 @@ title: How to create a site-to-site connection between Azure Stack Hub and Publi
 description: Create a site-to-site (S2S) VPN gateway connection from an on-premises network to UKCloud for Microsoft Azure 
```

Comment:
This doesn't sound right. We're making the connection between ASH and Azure so why is it talking about an on-premises network?